src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -36,22 +36,23 @@
  */
 class GetSetMethodNormalizer extends AbstractObjectNormalizer
 {
+    private static $reflectionCache = [];
     private static $setterAccessibleCache = [];
 
     /**
      * {@inheritdoc}
      */
-    public function supportsNormalization($data, string $format = null)
+    public function supportsNormalization($data, string $format = null, array $context = [])
```

Comment:
we took care of introducing the very same argument smoothly in upper branches (see https://github.com/symfony/serializer/blob/6.3/Normalizer/GetSetMethodNormalizer.php) so I'm not sure we can do it this way